src/advanced_inv.cpp

```diff
@@ -471,8 +488,10 @@ int advanced_inv_area::get_item_count() const
 
 void advanced_inv_area::init()
 {
+    offz = 0; // FIXME: merely a placeholder until Z levels
```

Comment:
This looks strange (even through it's just a placeholder), but why is off**z** set here, but off**x** and off**y** are (apparently) set somewhere else? All three offset values should be set at the same place, otherwise this will produce inconsistencies.